thinc/compat.py

```diff
@@ -31,7 +31,10 @@
     import torch
 
     has_torch = True
-    has_torch_gpu = torch.cuda.device_count() != 0
+    has_torch_cuda_gpu = torch.cuda.device_count() != 0
+    has_torch_mps = hasattr(torch, "has_mps") and torch.has_mps
+    has_torch_mps_gpu = has_torch_mps and torch.backends.mps.is_available()
```

Comment:
Is `has_torch_mps` by itself actually useful? (if `has_torch_mps_gpu` is `False`)
Asking to figure out whether we actually need to keep track of it?

Reply:
Removed `has_torch_mps`.